Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8617,28 +8770,65 @@ def main():
         reliability = DBotScoreReliability.get_dbot_score_reliability_from_str(reliability)
     else:
         raise Exception("Please provide a valid value for the Source Reliability parameter.")
-
     try:
+        results: List[CommandResults] = []
         if command == 'ip':
-            demisto_args = demisto.args()
-            ip = demisto_args.get('ip')
-            ret_value = ip_command(ip, reliability)
-            if ret_value:
-                return_results(ret_value)
+            ip = args.get('ip', '')
+            rate_limit_retry_count: int = arg_to_number(get_param_or_arg(
+                param_key='rate_limit_retry_count', arg_key='rate_limit_retry_count'), arg_name="rate_limit_retry_count") or RATE_LIMIT_RETRY_COUNT_DEFAULT
+            rate_limit_wait_seconds: int = arg_to_number(get_param_or_arg(
+                param_key='rate_limit_wait_seconds', arg_key="rate_limit_wait_seconds")) or RATE_LIMIT_WAIT_SECONDS_DEFAULT
+            rate_limit_errors_suppressed: bool = argToBoolean(get_param_or_arg(
+                param_key='rate_limit_errors_suppressed', arg_key='rate_limit_errors_suppressed')) or RATE_LIMIT_ERRORS_SUPPRESSEDL_DEFAULT
```

Comment:
I thought it would be easier to feed the arguments specific to the IP command to the function and makes it easier to test (rather than `mock.patch` each one of them inside the UT). I wrote a UT for it as well, `test_ip_command` .

Reply:
I understand but if the only thing it causes is the usage of mocks then let's revert it. The unit tests are not the core priority, the code clearness, and standards are more important.

Reply:
https://github.com/demisto/content/pull/27370/commits/40a6cbfe2e250f8b1e2ec36dbf65806792ad41e5